survey/migrations/0001_initial.py

```diff
@@ -64,24 +82,65 @@ class Migration(migrations.Migration):
             },
         ),
         migrations.CreateModel(
-            name='Answer',
+            name='SurveyAnswer',
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('created_at', models.DateTimeField(auto_now_add=True)),
                 ('modified_at', models.DateTimeField(auto_now=True)),
-                ('code', models.CharField(max_length=10, unique=True)),
+                ('answer', models.TextField(blank=True, default=None, null=True)),
+                ('answer_type', models.CharField(choices=[('check_box', 'Check Box'), ('date', 'Date'), ('description', 'Description'), ('image', 'Image'), ('location', 'Location'), ('number', 'Number'), ('single_option', 'Single Option'), ('multiple_option', 'Multiple Option')], max_length=15)),
```

Comment:
should we keep these enums as a variable somewhere? This may be used in both question and surveyanswer.

Reply:
Oops just noticed this is in migration